ollama/_types.py

```diff
@@ -333,7 +334,7 @@ class ModelDetails(SubscriptableBaseModel):
 
 
 class ListResponse(SubscriptableBaseModel):
-  class Model(BaseModel):
+  class Model(SubscriptableBaseModel):
```

Comment:
Didn't end up gutting this out - did a quick mock to see what it would look like and I think it's not worth the extra complexity just yet. I think there is some server and pattern work to be done before we can have a base class for model. 

I'd ideally want to be able to share this class a lot more freely with those consistencies. With that, I think it's okay for it to be out of scope for this and your PR.